variables.tf

```diff
@@ -14,6 +14,24 @@ variable "instance_name" {
   description = "Name of the security and compliance instance that will be provisioned by this module"
 }
 
+variable "provider_type_instance_name" {
+  type        = string
+  description = "The name for the provider_type instance"
+  default     = null
+}
+
+variable "attributes" {
+  type        = map(string)
+  description = "The attributes for connecting to the provider type instance"
+  default     = {}
+}
+
+variable "provider_type_id" {
+  type        = string
+  description = "The unique identifier of the provider type instance."
+  default     = null
+}
```

Comment:
Had a chat with SCC provider dev, they are going to add a data lookup to lookup this ID by name and region. So until that is implemented, I don't think we should implement this feature. Perhaps update the code with other review comments and mark PR as draft while we wait